models/quickbooks__general_ledger.sql

```diff
@@ -95,7 +96,7 @@ accounts as (
 adjusted_gl as (
     select
         gl_union.transaction_id,
-        row_number() over(partition by gl_union.transaction_id order by gl_union.transaction_date) as transaction_index,
+        index as transaction_index,
```

Comment:
Since the `transaction_index` can be repeating now, it may be beneficial to add a [surrogate key](https://github.com/dbt-labs/dbt-utils#surrogate_key-source) to identify uniqueness.

I am thinking a combination of `gl_union.transaction_id`, `gl_union.index`, and `accounts.name`. What are your thoughts?

Reply:
New changes added!